lib/ios/AirMaps/AIRMapPolylineRenderer.m

```diff
@@ -0,0 +1,252 @@
+//
+//  AIRMapPolylineRenderer.h
+//  mapDemo
+//
+//  Created by IjzerenHein on 13-11-21.
+//  Copyright (c) 2017 IjzerenHein. All rights reserved.
+//
+
+#import "AIRMapPolylineRenderer.h"
+
+@interface AIRMapPolylineRendererSegment : NSObject
+- (id)initWithPoint:(CGPoint)point color:(UIColor*)color;
+- (void) addPoint:(CGPoint)point color:(UIColor*)color;
+@property UIColor *color;
+@property CGMutablePathRef path;
+@property UIColor *color2;
```

Comment:
can we find a better name for `color2` ?

Reply:
Good catch. You're right, it would have been better to name these `startColor` and `endColor`, because this is exactly what they are. I'll make the changes.